client/src/app/stylesheets/App.css

```diff
@@ -1,10 +1,35 @@
-a, a:hover, a:focus, a:active {
-    text-decoration: none;
-    color:inherit;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```

Comment:
might be a way to do this through Theme.js instead of in the most parent css file? just for modularity's sake @ntonioibarrola